lib/model.js

```diff
@@ -214,13 +214,6 @@ _.assign(props, {
   mongodb_database_name: {
     type: 'string',
     default: undefined
-  },
-  /**
-   * Whether BSON values should be promoted to their JS type counterparts.
-   */
-  promote_values: {
-    type: 'boolean',
-    default: true
```

Comment:
Ok that makes sense but this.promote_values can't be set to undefined since it's ampersand model prop with a type boolean. It looked like setting it to `undefined` would make it default to `true`.

Also setting promoteValues can be done directly via Compass in current master by adding:
`state.connection.driver_options.db.promoteValues = false;`
above [line 433 on `src/app/index.js`](https://github.com/10gen/compass/blob/1a8acdb79cbbb62d581bc61c5b394b9ecc28eca0/src/app/index.js#L433)